subset/ping/ping.daqmodule

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "ping",
+    "startupScript": "test_ping"
```

Comment:
should be startup_script following other naming conventions

Reply:
I followed the json naming convention here but it's not ideal, I can convert to snake case.

Reply:
understood... JSON ends up being all over the place and really has nothing to do with JS anymore :-).  Generally I've found that snake_case is better than cAmel case and Java/JS just got it wrong.  Anyway -- my general guidance is if possible go with snake_case even if it's "wrong" (e.g. sometimes it shows up in Java code too...)